dds/DCPS/DataWriterImpl.cpp

```diff
@@ -1345,7 +1345,7 @@ DataWriterImpl::enable()
                                            qos_.reliability.max_blocking_time,
                                            n_chunks_,
                                            domain_id_,
-                                           get_topic_name(),
+                                           topic_name_.in(),
```

Comment:
Ok, found it, but that file needs some major updates related to svn, bug reporting, etc, maybe make an issue?